pkg/digger/digger.go

```diff
@@ -566,23 +566,24 @@ func cleanupTerraformOutput(nonEmptyOutput bool, planError error, stdout string,
 		startPos = 0
 	}
 
-	regex := regexp.MustCompile(regexStr)
-	matches := regex.FindStringSubmatch(stdout)
-	if len(matches) > 0 {
-		endPos = strings.Index(stdout, matches[0]) + len(matches[0])
+	if regexStr != nil {
+		regex := regexp.MustCompile(*regexStr)
+		matches := regex.FindStringSubmatch(stdout)
+		if len(matches) > 0 {
+			endPos = strings.Index(stdout, matches[0]) + len(matches[0])
+		}
 	}
 
 	return stdout[startPos:endPos]
 }
 
 func cleanupTerraformApply(nonEmptyPlan bool, planError error, stdout string, stderr string) string {
-	regex := `(Apply complete! Resources: [0-9]+ added, [0-9]+ changed, [0-9]+ destroyed.)`
-	return cleanupTerraformOutput(nonEmptyPlan, planError, stdout, stderr, regex)
+	return cleanupTerraformOutput(nonEmptyPlan, planError, stdout, stderr, nil)
 }
 
 func cleanupTerraformPlan(nonEmptyPlan bool, planError error, stdout string, stderr string) string {
-	regex := `(Plan: [0-9]+ to add, [0-9]+ to change, [0-9]+ to destroy.)`
-	return cleanupTerraformOutput(nonEmptyPlan, planError, stdout, stderr, regex)
+	regex := `───────────.+`
```

Comment:
what is this for?

Reply:
without it we also see the "plan saved" part which shouldn't be included

<img width="927" alt="Ekran Resmi 2023-05-10 15 24 22" src="https://github.com/diggerhq/digger/assets/1627972/f8559bad-82ba-43b5-a602-3eee88435da6">